api/authentication.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.exceptions import AuthenticationFailed
+from django.conf import settings
+
+
+class SingleTokenAuthentication(TokenAuthentication):
+    def authenticate(self, request) -> None:
+        request_token = request.headers["Authorization"].split(" ")[-1]
+
+        if request_token == settings.REST_API_CONTACT_SINGLE_TOKEN:
```

Comment:
Defining the token (for now) in the .env via settings makes sense. Why is there still an AuthToken.tokens model associated with users though? This thing. This could be something I don't understand about django-rest. Could you explain?


<img width="436" alt="Screenshot 2024-05-10 at 21 07 24" src="https://github.com/bank-green/bankgreen-django/assets/1609475/aff5afb3-84ad-4022-acf2-6891a7270a16">